idea/testData/joinLines/stringTemplate/simple.kt

```diff
@@ -0,0 +1,4 @@
+fun test1() {
```

Comment:
Some questions:
* does this work for more than 2 lines, e.g. 
```kotlin
val s = "one"
+ ", two"
+ ", three"
```
* does it work for non-strings, e.g.
```java
val s = "x=" +
0
```

Reply:
The Join Lines action joins the line where the caret is with the next line. It always acts on exactly two lines.

Reply:
Thanks

Reply:
Fixed